evap/locale/de/LC_MESSAGES/djangojs.po

```diff
@@ -0,0 +1,71 @@
+# SOME DESCRIPTIVE TITLE.
+# Copyright (C) YEAR THE PACKAGE'S COPYRIGHT HOLDER
+# This file is distributed under the same license as the PACKAGE package.
+# FIRST AUTHOR <EMAIL@ADDRESS>, YEAR.
+#
```

Comment:
weird default copyright header -- I'd just replace it with the header from `django.po`?


```suggestion
# EvaP translation
# This file is distributed under the same license as the EvaP project.
#
```